app/models/match.rb

```diff
@@ -64,26 +65,12 @@ def get_winner
     self.redGoal > self.blueGoal ? 'r' : 'b'
   end
 
-  def self.get_player_matches_json(player_id)
-    # return json of matches
-    matches = []
-    single_pm_records = PlayerMatch.where(:player_id => player_id)
-    single_pm_records.each do |pm|
-      matches.push(Match.where(:id => pm.match_id).take!)
-    end
-
-    match_json_arr = get_json_from_matches_array(matches)
-
-    puts JSON.pretty_generate(match_json_arr)
-    return JSON.pretty_generate(match_json_arr)
+  def self.get_player_matches(player)
+    get_matches_arr(player.matches)
   end
 
-  def self.get_all_matches_json
-    return JSON.pretty_generate(get_json_from_matches_array(Match.all))
-  end
-
-  def self.get_match_json(match)
-    return JSON.pretty_generate(get_json_from_match(match))
+  def self.get_all_matches_arr
+    return get_matches_arr(Match.all)
```

Comment:
don need the keyword `return`. The last line is always implicitly returned by ruby